include/bgfx/defines.h

```diff
@@ -492,6 +492,7 @@
 #define BGFX_CAPS_VERTEX_ID                       UINT64_C(0x0000000008000000) //!< Rendering with VertexID only is supported.
 #define BGFX_CAPS_VIEWPORT_LAYER_ARRAY            UINT64_C(0x0000000010000000) //!< Viewport layer is available in vertex shader.
 #define BGFX_CAPS_DRAW_INDIRECT_COUNT             UINT64_C(0x0000000020000000) //!< Draw indirect with indirect count is supported.
+#define BGFX_CAPS_FRAGMENT_BARYCENTRIC            UINT64_C(0x0000000040000000) //!< Fragment barycentric coordinates are available in fragment shader.
```

Comment:
This is generated file. You must add this to scripts/bgfx.idl, and then run `genie idl` in scripts directory. Also bump IDL version number.

Reply:
Done.